localisation/00_wc_artifact_localisation.csv

```diff
@@ -128,6 +130,8 @@ libram_of_righteousness_03;Libram of Righteousness;;Foliant der Rechtschaffenhei
 libram_of_righteousness_03_desc;This libram is a symbol of righteousness before the ideals of the holy light. Its owner proves to be a faithful student and righteous fighter for the light.;;Dieser Foliant ist ein Symbol der Rechtschaffenheit vor den Idealen des heiligen Lichts. Sein Besitzer erweist sich als treuer Sch�ler und rechtschaffener K�mpfer f�r das Licht.;;;;;;;;;;;x
 lights_vengeance;Light's Vengeance;;Lichtr�cher;;;;;;;;;;;x
 lights_vengeance_desc;Lordaeron's crownprince acquired this warhammer at his ceremony of admission in the Cathedral of Light in Stormwind, when he joined the ranks of the Silver Hand. Its wielder will find no peace until [Prev.GetHerHis] religious enemies lie dead before [Prev.GetHerHim].;;Lordaerons Kronprinz erhielt diesen Hammer, als er seine Initiation in der Kathedrale des Lichts in Stormwind abschloss und sich den Reihen der Silbernen Hand anschloss. Sein[Prev.GeteFemale] Tr�ger[Prev.GetinFemale] wird keinen Frieden finden, bis seine Feinde tot vor [Prev.Getihmihr] liegen.;;;;;;;;;;;x
+mantle_of_gorak_tul;Mantle of Gorak Tul;;Gorak Tuls Mantel;;;;;;;;;;;x
+mantle_of_gorak_tul_desc;A shimmering leather coat that has survived the centuries unscathed thanks to dark magic. Property of the one true ruler of Kul Tiras.;;Ein schimmernder, lederner Mantel, der die Jahrhunderte dank dunkler Magie unbeschadet �berstanden hat. Eigentum des einzig wahren Herrschers von Kul Tiras.;;;;;;;;;;;x
```

Comment:
I mean, the image you used looks more like a cloak than a coat, and it doesn't really look like leather, more like feathers woven together. Even in WoW where the armour is leather, it is a set of shoulderpads, not a cloak. Suggested a change as a result.
```suggestion
mantle_of_gorak_tul_desc;A shimmering cloak that has survived the centuries unscathed thanks to dark magic. Property of the one true ruler of Kul Tiras.;;Ein schimmernder, lederner Mantel, der die Jahrhunderte dank dunkler Magie unbeschadet �berstanden hat. Eigentum des einzig wahren Herrschers von Kul Tiras.;;;;;;;;;;;x
```

Reply:
why remove the leather part? in wow it is made of leather...

Reply:
fixed